src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -39,7 +39,7 @@ public EventCard(Event event, int displayedIndex) {
         date.setText(event.getDateString());
         time.setText(event.getTimeDisplayString());
         int noOfParticipants = event.getParticipants().size();
-        if (noOfParticipants <= 1) {
+        if (noOfParticipants == 1) {
```

Comment:
Actly this should be <= 1 I guess, cuz 0 participant also?

Reply:
I think it should be "0 participants", grammatically speaking

Reply:
Oh, then this ba, my ang moh not strong enough 😅